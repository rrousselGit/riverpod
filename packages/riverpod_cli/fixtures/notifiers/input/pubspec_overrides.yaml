dependency_overrides:
  flutter_riverpod: ^0.13.0
  hooks_riverpod: ^0.13.0
  riverpod: ^0.13.0
