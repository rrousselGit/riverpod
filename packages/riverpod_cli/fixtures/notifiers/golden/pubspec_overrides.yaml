dependency_overrides:
  flutter_riverpod: ^0.14.0
  hooks_riverpod: ^0.14.0
  riverpod: ^0.14.0
